src/viam/sdk/module/service.cpp

```diff
@@ -263,28 +262,9 @@ ModuleService::~ModuleService() {
     }
 }
 
-void ModuleService::add_api_from_registry_inlock_(API api, const std::lock_guard<std::mutex>&) {
```

Comment:
This was adding managed resource servers/resources from the registry to the module_. But, since all resource servers are now owned/maintained by the `Server`, this is no longer necessary.